bigtable/admin.go

```diff
@@ -2371,6 +2474,29 @@ func (ac *AdminClient) UpdateBackup(ctx context.Context, cluster, backup string,
 	return err
 }
 
+// UpdateBackupHotToStandardTime updates the HotToStandardTime of a hot backup.
+func (ac *AdminClient) UpdateBackupHotToStandardTime(ctx context.Context, cluster, backup string, hotToStandardTime *time.Time) error {
```

Comment:
Could we use `HotBackupOptions` here to eliminate another `*time.Time` parameter

Reply:
Having the same options for create and update can complicate this later. 
Updated the PR to add 2 separate methods similar to other implementation:

https://github.com/googleapis/google-cloud-go/blob/8109157cb2bfb700fde04361e0fa7c1345608fce/bigtable/admin.go#L487-L507